src/bin/pg_autoctl/cli_perform.c

```diff
@@ -190,13 +204,25 @@ cli_perform_failover_getopts(int argc, char **argv)
 		exit(EXIT_CODE_BAD_ARGS);
 	}
 
-	cli_common_get_set_pgdata_or_exit(&(options.pgSetup));
-
-	if (!keeper_config_set_pathnames_from_pgdata(&(options.pathnames),
-												 options.pgSetup.pgdata))
+	/* when we have a monitor URI we don't need PGDATA */
+	if (cli_use_monitor_option(&options))
 	{
-		/* errors have already been logged */
-		exit(EXIT_CODE_BAD_ARGS);
+		if (!IS_EMPTY_STRING_BUFFER(options.pgSetup.pgdata))
+		{
+			log_warn("Given --monitor URI, the --pgdata option is ignored");
```

Comment:
Is this really what we want? Other options are hard error or make `--pgdata` win.

Also does the same hold for setting `PG_AUTOCTL_MONITOR` and then using `--pgdata`?

Reply:
The other commands also prefer using --monitor over --pgdata, I think we're good here?